frontend/packages/dev-console/integration-tests/features/monitoring/dashboard-display.feature

```diff
@@ -1,67 +1,115 @@
+              And user is on Observe page
              When user clicks on "Dashboard" tab
-              And user selects the workload "parks-test" from the dropdown
-             Then user will see the CPU Usage on Dashboard tab
-              And user will see the Memory Usage on Dashboard tab
-              And user will see Receive Bandwidth on Dashboard tab
-              And user will see Transmit Bandwidth on Dashboard tab
-              And user will see the Rate of Received Packets on Dashboard tab
-              And user will see the Rate of Transmitted Packets on Dashboard tab
-              And user will see the Rate of Received Packets Dropped on Dashboard tab
-              And user will see the Rate of Transmitted Packets Dropped on Dashboard tab
+              And user clicks on Dashboard dropdown
+              And user clicks on "Kubernetes / Compute Resources / Namespace (Workloads)" option
+              And user clicks on Type dropdown
+              And user selects "deployment" option from the dropdown
+             Then user will see "CPU Usage" chart
+              And user will see "CPU Quota" chart
+              And user will see "Memory Usage" chart
+              And user will see "Memory Quota" chart
+              And user will see "Current Network Usage" chart
+              And user will see "Bandwidth" charts
+              And user will see "Rate of Packets" charts
+              And user will see "Rate of Packets Dropped" charts
 
 
         @regression
-        Scenario: Dashboard tab on the Monitoring page for all workloads: M-02-TC03
-            Given user is at Add page
-              And user has created workload "parks-test-1" with resource type "Deployment"
-              And user opened the url of the workload "parks-test-1" in topology page
-              And user is on Monitoring page
+        Scenario: Dashboard tab on the Observe page with Kubernetes / Compute Resources / Namespace (Pods) option selected: M-02-TC03
+            Given user is on Observe page
+             When user clicks on "Dashboard" tab
+              And user clicks on Dashboard dropdown
+              And user clicks on "Kubernetes / Compute Resources / Namespace (Pods)" option
+             Then user will see "CPU Utilisation (from requests)" chart
+              And user will see "CPU Utilisation (from limits)" chart
+              And user will see "Memory Utilisation (from requests)" chart
+              And user will see "Memory Utilisation (from limits)" chart
+              And user will see "CPU Usage" chart
+              And user will see "CPU Quota" chart
+              And user will see "Memory Usage" chart
+              And user will see "Memory Quota" chart
+              And user will see "Current Network Usage" chart
+              And user will see "Bandwidth" charts
+              And user will see "Rate of Packets" charts
+              And user will see "Rate of Packets Dropped" charts
```

Comment:
Modify the charts to chart

Reply:
More than one chart appearing for Rate of Packets Dropped